packages/calendar/src/compute.js

```diff
@@ -94,21 +126,29 @@ const monthPathAndBBox = ({
     direction,
     originX,
     originY,
+    fromDate,
+    toDate,
+    exact,
 }) => {
     // first day of next month
     const t1 = new Date(date.getFullYear(), date.getMonth() + 1, 0)
 
+    const startDate = calculateReturnDate(fromDate, timeYear(date), exact)
+    const elapsedMonths = getElapsedMonths(startDate, date)
+    const lastDate = calculateReturnDate(fromDate, timeYear(t1), exact)
+    // if the toDate is not the last of the months, we do not need to paint the path
+    const lastDayOfMonth = new Date(toDate.getFullYear(), toDate.getMonth() + 1, 0)
     // ranges
-    const firstWeek = timeWeek.count(timeYear(date), date)
-    const lastWeek = timeWeek.count(timeYear(t1), t1)
+    let firstWeek = timeWeek.count(startDate, date)
```

Comment:
Why changing from `const` to `let` ? I don't see it being updated anywhere

Reply:
I was changing while i was developing and forgot to revert it